docs/application/dotnet/get-started/wearable/first-app.md

```diff
@@ -1,183 +1,124 @@
 the Tizen platform-specific project is generated.
 
-The `.cs` file in the portable project already contains simple
-Xamarin.Forms code that makes a basic UI.
+The `.cs` file in the portable project already contains simple Xamarin.Forms code that makes a basic UI.
 
-## Building Your Application
+## Build Your Application
 
-After you have created the application project, you can implement the
-required features. In this example, only the default features from the
-project template are used, and no code changes are required.
+After you have created the application project, you can implement the required features. In this example, only the default features from the project template are used, and no code changes are required.
 
-When your application code is ready, build the application. The building
-process performs a validation check and compiles your files. You must
-sign the application package with an author certificate when building
-the application. If you have not yet registered a Tizen certificate in
-Visual Studio, see [Certificate
-Manager](../../../vstools/tools/certificate-manager.md).
+When your application code is ready, build the application. The building process performs a validation check and compiles your files. You must sign the application package with an author certificate when building the application. If you have not yet registered a Tizen certificate in Visual Studio, see [Certificate Manager](../../../vstools/tools/certificate-manager.md).
 
-There are 2 different ways to build the application:
+There are two different ways to build the application:
 
--   In the Visual Studio menu, select **Build &gt; Build Solution**.
--   In the **Solution Explorer** view, right-click the solution name and
-    select **Build**.
+-   In the Visual Studio menu, select **Build \> Build Solution**.
+-   In the **Solution Explorer** view, right-click the solution name and select **Build**.
 
-Tizen .NET applications are always deployed as installed packages. The
-package files have the `.tpk` file extension, and the process of generating a package is controlled by the [manifest
-file](../../../vstools/tools/manifest-editor.md). The Visual Studio
-template generates the manifest file (`tizen-manifest.xml`) to the top
-level of the &lt;projectname&gt;.Tizen project (if you create projects
-with mobile, TV, or wearable profiles, a separate manifest file is
-generated for each profile).
+Tizen .NET applications are always deployed as installed packages. The package files have the `.tpk` file extension, and the process of generating a package is controlled by the [manifest file](../../../vstools/tools/manifest-editor.md). The Visual Studio template generates the manifest file (`tizen-manifest.xml`) to the top level of the \<projectname\>.Tizen project (if you create projects with mobile, TV, or wearable profiles, a separate manifest file is generated for each profile).
```

Comment:
Referencing the update in https://github.com/Samsung/tizen-docs/pull/1018 : 

```suggestion
Tizen .NET applications are always deployed as installed packages. The package files have the `.tpk` file extension, and the process of generating a package is controlled by the [manifest file](../../../vstools/tools/manifest-editor.md). The Visual Studio template generates the manifest file (`tizen-manifest.xml`) to the top level of the \<projectname\>.Tizen project (if you create projects with Mobile, TV, or Wearable profiles, a separate manifest file is generated for each profile).
```

Reply:
I restored the previous one.